ui/src/components/Secrets.vue

```diff
@@ -217,7 +217,7 @@ export default {
         response = await fetch(`${apiUrl}/namespaces`);
 
         let availableNamespaces = await response.json();
-        this.namespaces = JSON.parse(availableNamespaces);
+        this.namespaces = availableNamespaces;
```

Comment:
Would it make sense to directly send the result of `await response.json()` into `this.namespaces`? (same for the above config assignment)